Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -2,32 +2,352 @@ import Foundation
+      baseStackView.rightAnchor.constraint(equalTo: containerMargins.rightAnchor),
+      baseStackView.topAnchor.constraint(equalTo: containerMargins.topAnchor)
+    ])
+
+    // Pledge button
+    _ = pledgeButton
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+
+    let topConstraint = pledgeButton.topAnchor.constraint(equalTo: pledgeButtonLayoutGuide.topAnchor)
+    _ = topConstraint
+      |> \.priority .~ .defaultLow
+      |> \.isActive .~ true
+
+    let contentMargins = self.contentView.layoutMarginsGuide
+
+    NSLayoutConstraint.activate([
+      pledgeButton.leftAnchor.constraint(equalTo: contentMargins.leftAnchor),
+      pledgeButton.rightAnchor.constraint(equalTo: contentMargins.rightAnchor),
+      // swiftlint:disable:next line_length
+      pledgeButton.bottomAnchor.constraint(lessThanOrEqualTo: contentMargins.bottomAnchor),
+      // swiftlint:disable:next line_length
+      pledgeButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Styles.minTouchSize.height)
+    ])
+
+    // Pledge button layout anchor
+    NSLayoutConstraint.activate([
+      pledgeButtonLayoutGuide.bottomAnchor.constraint(equalTo: containerMargins.bottomAnchor),
+      pledgeButtonLayoutGuide.leftAnchor.constraint(equalTo: containerMargins.leftAnchor),
+      pledgeButtonLayoutGuide.rightAnchor.constraint(equalTo: containerMargins.rightAnchor),
+      // swiftlint:disable:next line_length
+      pledgeButtonLayoutGuide.topAnchor.constraint(equalTo: baseStackView.bottomAnchor, constant: Styles.grid(3)),
+      pledgeButtonLayoutGuide.heightAnchor.constraint(equalTo: pledgeButton.heightAnchor)
+    ])
+  }
+
+  fileprivate func load(items: [String]) {
+    self.includedItemsStackView.subviews.forEach { $0.removeFromSuperview() }
+
+    let includedItemViews = items.enumerated().map { (index, item) -> [UIView] in
+      let label = UILabel()
+        |> baseRewardLabelStyle
+        |> sectionBodyLabelStyle
+        |> \.text .~ item
+
+      let separator = UIView()
+        |> separatorStyle
+      separator.heightAnchor.constraint(equalToConstant: 1).isActive = true
+
+      var itemViews: [UIView] = [label]
+
+      if index != items.endIndex - 1 {
+        itemViews.append(separator)
+      }
+
+      return itemViews
+    }.flatMap { $0 }
+
+    let allItemViews = [self.includedItemsTitleLabel] + includedItemViews
+
+    _ = (allItemViews, self.includedItemsStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+  }
+
+  internal func configureWith(value: (Project, Either<Reward, Backing>)) {
+    self.viewModel.inputs.configureWith(project: value.0, rewardOrBacking: value.1)
+  }
+
+  // MARK: - Selectors
+
+  @objc func pledgeButtonTapped() {
+    self.viewModel.inputs.pledgeButtonTapped()
+  }
+
+  @objc func rewardCardTapped() {
+    self.viewModel.inputs.rewardCardTapped()
+  }
+}
+
+// MARK: - Styles
+
+private let baseRewardLabelStyle: LabelStyle = { label in
+  label
+    |> \.numberOfLines .~ 0
```

Comment:
Are you suggesting doing: `|> \.numberOfLines .~ Int(0)` here? The compiler isn't currently showing a type-check warning here so it doesn't seem necessary. 🤔

Reply:
I'm sorry, I was referring to the other rows (basically anything that's not a primitive like `lineBreakMode`, `backgroundColor`, etc.)

Reply:
Cool beans..this is unfortunately CPU dependent so it might be fine on some machines but not as performant on other machines...I thought we have agreed to this earlier when working on the checkout but maybe it got missed in conversation ¯\_(ツ)_/¯